src/PerfView/PerfViewData.cs

```diff
@@ -4153,6 +4154,72 @@ protected internal override StackSource OpenStackSourceImpl(string streamName, T
                     stackSource.AddSample(sample);
                 }
             }
+            else if (streamName == ".NET Native CCW Ref Count")
+            {
+                var objectTypeMap = new Dictionary<long, long>(1000);
+                var interopTraceEventParser = new InteropTraceEventParser(eventSource);
+                interopTraceEventParser.AddCallbackForEvent<TaskCCWCreationArgs>(null, args =>
+                {
+                    if(!objectTypeMap.ContainsKey(args.objectID))
+                        objectTypeMap.Add(args.objectID, args.targetObjectIDType);
+                });
+                interopTraceEventParser.AddCallbackForEvents<TaskCCWQueryRuntimeClassNameArgs>(args =>
```

Comment:
Do you have a reason for including TaskCCWQueryRuntimeClassName in the stacks?  Is it just that it has a string name?    At the very least you should insure that sample.Metric is set to 0 (otherwise it is whatever got left over from the last event).  

But I am thinking that mostly it is not clear that it useful to include it at all.   (If we do, it should be included in the information remembered by your objectTypeMap (but now it is an objectInfoMap).  I would drop it completely for now.

Reply:
okay, if we would get this info from pdb